mls-rs-uniffi/src/config/group_state.rs

```diff
@@ -28,9 +31,13 @@ impl From<EpochRecord> for mls_rs_core::group::EpochRecord {
     }
 }
 
-#[cfg_attr(not(mls_build_async), maybe_async::must_be_sync)]
+// When building for async, uniffi::export has to be applied _after_
+// maybe-async's injection of the async trait. When building for sync,
+// the order has to be the opposite.
```

Comment:
I haven't looked into why this is — I suspect I could figure it out by staring at `cargo expand` a bit.

Reply:
Outer macros are applied before inner ones, so it's actually the other way around:
- For async, `uniffi::export` needs to be applied first (apparently [because](https://github.com/mozilla/uniffi-rs/pull/1981#discussion_r1482139339) `uniffi::export` cannot deal with the code `#[async_trait]` generates).
- For sync, `uniffi::export` needs to see the code after `maybe_async` has transformed it to remove the async-related keywords.

Reply:
This was merged but I think the comment got the order wrong and needs amending.

Reply:
Done in #131.

Reply:
Thanks @stefunctional!

Reply:
You're welcome. Thank you for your work on this!